### DISCUSSION TITLE
Cider App

### DISCUSSION THREAD

**Discussion by JaxTer193** [ciderapp/Cider#902] [Rejected Features]
An app for mobile systems being able to use Cider instead of apple music on either Android or iPhone Operating System (iOS)/ iPad Operating System (iPadOS)

**Comment by Amaru8** [comment]
**Hi @JaxTer193!** Most likely, your idea won’t happen this year, as it would require a complete rewrite of Cider’s codebase.

**Reply by yazninja** [reply]
But also, based on how musckit on android is. All processes with apple servers are still handled by the AM app, so aside from having cider mobile , you hypothetically would also need to open apple music, or have it downloaded at least
